torch_np/_funcs_impl.py

```diff
@@ -1945,6 +1945,61 @@ def histogram(
     return h, b
 
 
+def histogramdd(
+    sample,
+    bins=10,
+    range: Optional[ArrayLike] = None,
+    normed=None,
+    weights: Optional[ArrayLike] = None,
+    density=None,
+):
+    # have to normalize manually because `sample` interpretation differs
+    # for a list of lists and a 2D array
+    if normed is not None:
+        raise ValueError("normed argument is deprecated, use density= instead")
+
+    from ._normalizations import normalize_array_like, normalize_seq_array_like
+
+    if isinstance(sample, (list, tuple)):
+        sample = normalize_array_like(sample).T
+    else:
+        sample = normalize_array_like(sample)
+
+    sample = torch.atleast_2d(sample)
+
+    if not (sample.dtype.is_floating_point or sample.dtype.is_complex):
+        sample = sample.double()
+
+    # bins is either an int, or a sequence of ints or a sequence of arrays
+    bins_is_array = not (
+        isinstance(bins, int) or builtins.all(isinstance(b, int) for b in bins)
+    )
+    if bins_is_array:
+        bins = normalize_seq_array_like(bins)
+        bins_dtypes = [b.dtype for b in bins]
+        bins = [_util.cast_if_needed(b, sample.dtype) for b in bins]
+
+    if range is not None:
+        range = range.flatten().tolist()
+
+    if weights is not None:
+        # range=... is required : interleave min and max values per dimension
+        mm = sample.amin(0), sample.amax(0)
```

Comment:
nit use `torch.aminmax`.

Reply:
Thanks! Done.
An unexpected twist is that keyword-only arguments look great, until `t.amin(0)` works but `t.aminmax(0)` doesn't :-).